apps/custom-ui-floating-window-demo/pages/api/getContact.js

```diff
@@ -0,0 +1,68 @@
+import { getCookie } from "cookies-next";
+import { PersonsApi } from "pipedrive";
+import logger from "../../shared/logger";
+import { initAPIClient } from "../../shared/oauth";
+const log = logger("Get Contact API 📕");
+/**
+ * Get the current session
+ * Obtain the person by ID / Number
+ * Get associated deals of that person (if available)
+ * Return the response
+ */
+export default async function handler(req, res) {
```

Comment:
this function is quite complicated to read, maybe you can split it into smaller functions and call them here?